cfme/tests/openstack/infrastructure/test_provider.py

```diff
@@ -18,10 +19,15 @@ def test_api_port(provider):
     api_port = int(view_details.entities.summary('Properties').get_text_of('API Port'))
     assert api_port == port, 'Invalid API Port'
 
+
 def test_credentials_quads(provider):
     view = navigate_to(provider, 'All')
     prov_item = view.entities.get_entity(name=provider.name, surf_pages=True)
-    assert prov_item.data.get('creds') and 'checkmark' in prov_item.data['creds']
+    valid_message = 'Authentication credentials are valid'
+    if current_version() >= '5.10':
+        assert valid_message in prov_item.data['quad']['bottomRight']['tooltip']
```

Comment:
I am suspect about this; can you please check for `creds` attribute for `5.10`

Reply:
There is no `creds` for 5.10

For example:
CFME 5.10
ipdb> dict(prov_item.data)
```
{'templates': u'5', 'datastores': u'0', 'quadicon': None, 'quad': {u'middle': None, u'bottomLeft': {u'fileicon': u'/assets/svg/vendor-openstack_infra-3aa93728245a70177256ca7f8bfb3b2bd69eb0b039e414ae3fe48c121226946e.svg', u'tooltip': u'Infrastructure Provider (OpenStack)'}, u'topRight': {u'text': u'0', u'tooltip': u'0 Virtual Machines'}, u'bottomRight': {u'fonticon': u'pficon pficon-ok', u'tooltip': u'Authentication credentials are valid'}, u'topLeft': {u'text': u'5', u'tooltip': u'5 Hosts'}}, 'id': u'1', 'evm_zone': u'default', 'vcores': u'32', 'gtltype': u'grid', 'name': u'uc_ci', 'region': u'Region 0', 'hostname': u'192.168.24.2', 'vms': u'0', '1': None, '0': None, 'hosts': u'5', 'memory': u'60 GB', 'discovered_ip_address': u'192.168.24.2', 'type': u'OpenStack Platform Director', 'long_id': u'1'}
```

CFME 5.9
ipdb> dict(prov_item.data)
```
{'templates': u'0', 'datastores': u'0', 'vendor': '/assets/svg/vendor-openstack_infra-3aa93728245a70177256ca7f8bfb3b2bd69eb0b039e414ae3fe48c121226946e.svg', 'no_host': 0, 'quadicon': u'<div class="quadicon" id="quadicon_13"><div class="flobj "><img src="/assets/layout/base-bde4d1415aa8c028fab2979574be568b3a8127001ae24babb8abd48c375a2872.svg" alt="Base" width="72" height="72" /></div><div class="flobj a72"><p style="">0</p></div><div class="flobj b72"><p>0</p></div><div class="flobj c72"><img src="/assets/svg/vendor-openstack_infra-3aa93728245a70177256ca7f8bfb3b2bd69eb0b039e414ae3fe48c121226946e.svg" alt="Vendor openstack infra" width="72" height="72" /></div><div class="flobj d72"><img src="/assets/100/checkmark-c17d7edb7937f444f6d79e59908bb8cb8b1534a2cae5e56f753cc23a1739ddee.png" alt="Checkmark" width="72" height="72" /></div><div class="flobj"><a title="Name: uc_ci | Hostname: 192.168.24.2 | Refresh Status: Error | Authentication Status: Valid" href="/ems_infra/13"><img border="0" src="/assets/layout/reflection-1601f9e02c2e0a1c1d2a4229f679746a3109dfc9d7456c55236a57f3a737728b.png" alt="Reflection" width="72" height="72" /></a></div></div>', 'id': u'13', 'creds': '/assets/100/checkmark-c17d7edb7937f444f6d79e59908bb8cb8b1534a2cae5e56f753cc23a1739ddee.png', 'evm_zone': u'default', 'vcores': u'', 'gtltype': u'grid', 'name': u'uc_ci', 'region': u'Region 0', 'hostname': u'192.168.24.2', 'vms': u'0', '1': None, '0': None, 'hosts': u'0', 'memory': u'', 'discovered_ip_address': u'', 'type': u'OpenStack Platform Director', 'long_id': u'13'}
```